File: chapter02/backend-app/src/backend-app.ts
import restana from "restana";
import cors from 'cors'
const service = restana();
service.use(cors())
const port = 3000;
service.get("/hi", async (req, res) =>
    res.send({
        success: true,
        message: "hello world!",
        timestamp:Date.now(),
        request: `${req.headers.host}${req.url}`,
    })
);
console.log(`listening on ${port}`);
service.start(port);

File: chapter02/backend-python/backend-app.py
from flask import Flask,request
from datetime import datetime
app = Flask(__name__)
port=3000
@app.route("/hi")
def hello_world():
    result={
        'success':True,
        'message': "hello world!",
        'timestamp':datetime.now().timestamp(),
        'request': f"{request.url}",
    }
    return result
if __name__=='__main__':
    app.run(port=port)